observer/circularQueueNodesProvider_test.go

```diff
@@ -23,6 +23,9 @@ func getDummyConfig() config.Config {
 				ShardId: 1,
 			},
 		},
+		GeneralSettings: config.GeneralSettingsConfig{
+			NumberOfShards: 2,
```

Comment:
should it be 2 or 3?

Reply:
not relevent, tests only, both 2 or 3 should work the same.. added 2 as Observers config only have 2 shards

Reply:
ok